Sources/TinkoffInvestSDK/SDK/TinkoffInvestSDK.swift

```diff
@@ -8,25 +8,32 @@
 public class TinkoffInvestSDK {
 
     // Properties
+    private let appName: String
+
     private let commonTokenProvider: TinkoffInvestTokenProvider
+    private let sandboxTokenProvider: TinkoffInvestTokenProvider
 
     // MARK: - Initialization
 
-    public init(tokenProvider: TinkoffInvestTokenProvider) {
+    public init(appName: String, tokenProvider: TinkoffInvestTokenProvider, sandbox sandboxTokenProvider: TinkoffInvestTokenProvider) {
```

Comment:
Actually [`x-app-name`](https://tinkoff.github.io/investAPI/grpc/) is kind of optional parameter for a regular SDK user, and basically it would be better at least set `"JohnReeze.TinkoffInvestSwiftSDK"` as default parameter in order to simplify SDK initialization.